### PR TITLE
Initial boiler

### PR DESCRIPTION
**Pull request by Juphex**

### CODE REVIEW

File: marimbabot_audio/package.xml [hunk 1]
```diff
@@ -0,0 +1,65 @@
+<?xml version="1.0"?>
+<package format="2">
+  <name>marimbabot_audio</name>
+  <version>0.0.0</version>
+  <description>The marimbabot_audio package</description>
```
Comment (Flova) [inline marimbabot_audio/package.xml:5]: Maybe a better description

File: marimbabot_audio/package.xml [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+
+  <!-- One maintainer tag required, multiple allowed, one person per tag -->
+  <!-- Example:  -->
+  <!-- <maintainer email="jane.doe@example.com">Jane Doe</maintainer> -->
+  <maintainer email="root@todo.todo">root</maintainer>
```
Comment (Flova) [inline marimbabot_audio/package.xml:10]: Maybe the email of somebody that wanted to do audio

File: marimbabot_audio/package.xml [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+
+
+  <!-- One license tag required, multiple allowed, one license per tag -->
+  <!-- Commonly used license strings: -->
+  <!--   BSD, MIT, Boost Software License, GPLv2, GPLv3, LGPLv2.1, LGPLv3 -->
+  <license>TODO</license>
```
Comment (Flova) [inline marimbabot_audio/package.xml:16]: We need to talk about licensing. Do we want to go with Apache License, Version 2.0?

File: marimbabot_audio/package.xml [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+
+
+  <!-- Url tags are optional, but multiple are allowed, one per tag -->
```
Comment (Flova) [inline marimbabot_audio/package.xml:19]: Maybe remove examples

File: marimbabot_bringup/package.xml [hunk 5]
```diff
@@ -0,0 +1,65 @@
+<?xml version="1.0"?>
+<package format="2">
+  <name>marimbabot_bringup</name>
+  <version>0.0.0</version>
+  <description>The marimbabot_bringup package</description>
```
Comment (Flova) [inline marimbabot_bringup/package.xml:5]: Same here

File: marimbabot_bringup/package.xml [hunk 6]
```diff
@@ -0,0 +1,65 @@
...
+
+  <!-- One maintainer tag required, multiple allowed, one person per tag -->
+  <!-- Example:  -->
+  <!-- <maintainer email="jane.doe@example.com">Jane Doe</maintainer> -->
+  <maintainer email="root@todo.todo">root</maintainer>
```
Comment (Flova) [inline marimbabot_bringup/package.xml:10]: And here

File: marimbabot_bringup/package.xml [hunk 7]
```diff
@@ -0,0 +1,65 @@
...
+
+
+  <!-- One license tag required, multiple allowed, one license per tag -->
+  <!-- Commonly used license strings: -->
+  <!--   BSD, MIT, Boost Software License, GPLv2, GPLv3, LGPLv2.1, LGPLv3 -->
+  <license>TODO</license>
```
Comment (Flova) [inline marimbabot_bringup/package.xml:16]: And here

File: marimbabot_planning/package.xml [hunk 8]
```diff
@@ -0,0 +1,65 @@
+<?xml version="1.0"?>
+<package format="2">
+  <name>marimbabot_planning</name>
+  <version>0.0.0</version>
+  <description>The marimbabot_planning package</description>
```
Comment (Flova) [inline marimbabot_planning/package.xml:5]: The same things from the other package.xml files apply also to this one.

File: marimbabot_vision/package.xml [hunk 9]
```diff
@@ -0,0 +1,65 @@
+<?xml version="1.0"?>
+<package format="2">
+  <name>marimbabot_vision</name>
+  <version>0.0.0</version>
+  <description>The marimbabot_vision package</description>
```
Comment (Flova) [inline marimbabot_vision/package.xml:5]: +1

File: marimbabot_audio/package.xml (hunk 3 above)
Comment (Juphex) [reply marimbabot_audio/package.xml:16]: Apache seems to be the way to go

File: marimbabot_audio/package.xml (hunk 2 above)
Comment (Juphex) [reply marimbabot_audio/package.xml:10]: depending on whether the person wants to have their email publically avaiblabe yes we could do that

Comment (Juphex) [reply marimbabot_audio/package.xml:10]: depending on whether the person wants to have their email publically avaiblabe

Comment (Flova) [reply marimbabot_audio/package.xml:10]: The mail is exposed either way due to git

Comment (Juphex) [reply marimbabot_audio/package.xml:10]: afaik it is not. You can set "Keep my email address private" inside the e-mail settings. We should figure this out within the group

Comment (Flova) [reply marimbabot_audio/package.xml:11]: This setting adds a dummy mail to the commit in your case `Juphex <20795522+Juphex@users.noreply.github.com>`. Be we can discuss this later on.

**Review by Flova — APPROVED** [review]

**Pull request merged**